packages/keyring-controller/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@metamask/keyring-controller",
+  "version": "0.0.0",
+  "description": "Stores identities seen in the wallet and manages interactions such as signing",
+  "keywords": [
+    "MetaMask",
+    "Ethereum"
+  ],
+  "homepage": "https://github.com/MetaMask/controllers/tree/main/packages/keyring-controller#readme",
+  "bugs": {
+    "url": "https://github.com/MetaMask/controllers/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/MetaMask/controllers.git"
+  },
+  "license": "MIT",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist/"
+  ],
+  "scripts": {
+    "changelog:validate": "../../scripts/validate-changelog.sh",
+    "doc": "../../scripts/doc.sh",
+    "test": "jest",
+    "test:watch": "jest --watch"
+  },
+  "dependencies": {
+    "@keystonehq/metamask-airgapped-keyring": "^0.6.1",
+    "@metamask/base-controller": "workspace:~",
+    "@metamask/controller-utils": "workspace:~",
+    "@metamask/message-manager": "workspace:~",
+    "@metamask/preferences-controller": "workspace:~",
+    "async-mutex": "^0.2.6",
+    "eth-keyring-controller": "^7.0.2",
+    "eth-sig-util": "^3.0.0",
+    "ethereumjs-util": "^7.1.4",
+    "ethereumjs-wallet": "^1.0.1"
+  },
+  "devDependencies": {
+    "@ethereumjs/common": "^2.6.4",
+    "@ethereumjs/tx": "^3.5.2",
```

Comment:
Previously this was
```suggestion
    "@ethereumjs/tx": "^3.2.1",
```

Reply:
Fixed here: https://github.com/MetaMask/controllers/pull/831/commits/b23a926a568ee825a72c75bd3ef40fc0b8156106